### PR TITLE
Update Mothership docs and User Guide for large Composer update

### PR DESCRIPTION
**Pull request by heliosfa**
This change updates the docs to reflect the changes that are being introduced on the Orchestrator's `FEATURE-0167-buffering_softswitch` branch.

Specifically, 
* all instances of `std::pair<uint32_t, P_Pkt_t>` in the docs that relate to `PKTS` messages or their input/output method chains are replaced with `P_Addr_Pkt_t`
* The Mothership now uses device indexes to lookup device addresses for log packets rather than the usurped pin addr.
* The User Guide has been updated with some useful Composer commands.

### CODE REVIEW

**Comment by mvousden** [comment]
Just so you know, I'm holding off on the Orchestrator PR before this one.

**Comment by heliosfa** [comment]
AS expected, hence why no review has been requested ;) I have another small update I need to make to these docs before I send it live for review as well.

**Review by mvousden — CHANGES_REQUESTED** [review]
This all looks fine to me, but I think either one of us should update the user guide with the Composer command set before we merge this all in :sweet_potato: .

**Comment by heliosfa** [comment]
> This all looks fine to me, but I think either one of us should update the user guide with the Composer command set before we merge this all in 🍠 .

Agreed, it needs to be one big merge of everything to bring it all up to date!

**Review by mvousden — APPROVED** [review]
Top brass. Thank you!

**Pull request merged**
